tests/mapper/test_full_batch_generators.py

```diff
@@ -356,32 +356,42 @@ def generator_flow(
         return A_dense, tind, y
 
     def test_generator_flow_notargets(self):
-        link_ids = list(self.G.edges())[:3]
+        sources, targets, _, _ = self.G.edge_arrays()
```

Comment:
If we're going with the approach of leaving `graph.edges` the same as it originally was and just adding a new `graph.edge_arrays` method, I think it might be better to just leave a lot of these tests as they originally were. In particular, all the tests that are testing `flow` methods it's probably more likely they'll continue to be called with `edges` in the original form?